cmd/a2o-migrate/cli/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"github.com/docker/docker/cmd/a2o-migrate"
+)
+
+var ( // auto generated on build
+	GitVersion = "undefined"
+	BuildTime  = "undefined"
+)
+
+func main() {
+	a2omigrate.GitVersion = GitVersion
+	a2omigrate.BuildTime = BuildTime
```

Comment:
Why do we override them here instead of setting directly those variables with build flags?

Reply:
I guess this is leftover of the restructure... `cmd/a2o-migrate` is built in a way that it can be compiled into github.com/balena-os/balena-engine-cli and it also has the `GitVersion`/`BuildTime` variables [here](https://github.com/balena-os/balena-engine/blob/a2o-migrate/cmd/a2o-migrate/a2o-migrate.go#L13-L15)

I wanted to be able to build it as a standalone tool using go modules though, which is also the reason for those [overrides](https://github.com/balena-os/balena-engine/blob/a2o-migrate/cmd/a2o-migrate/go.mod#L13-L17) and the `vendor/`

Do you have any suggestions what to do with that?

Reply:
Cannot we just set the `a2omigrate` package name on the variables we set from the Makefile?
https://github.com/balena-os/balena-engine/pull/168/files#diff-54936c3e2e129070063e5d3725547463R3

Reply:
done